packages/@react-native-windows/cli/src/e2etest/__snapshots__/projectConfig.test.ts.snap

```diff
@@ -139,14 +139,6 @@ Object {
 
 exports[`useWinUI3=true in react-native.config.js, useWinUI3=false in ExperimentalFeatures.props 1`] = `
 "<?xml version=\\"1.0\\" encoding=\\"utf-8\\"?>
-<packages>
-  <package id=\\"Microsoft.ReactNative.Cxx\\" version=\\"1.0.0\\" targetFramework=\\"native\\"/>
```

Comment:
Do we need to test the VCXPROJ file instead?
Note, `packages\@react-native-windows\cli\src\e2etest\projects\WithWinUI3\windows\WithWinUI3\WithWinUI3.vcxproj` currently only references `$(WinUIPackageName)`, version `$(WinUIPackageVersion)`.

@asklar @NickGerleman

Reply:
the autolinking CLI had logic to rewrite packages.config - I haven't checked but you should remove that logic and simplify things. 
Instead of this test, we should have a test that setting UseWinUI3 = true in ExperimentalFeatures.props of a new app created with the CLI, will restore the WinUI3 package and not the WinUI2 package